velox/row/tests/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ target_link_libraries(
   velox_exec_test_lib
   velox_functions_lib
   velox_functions_prestosql
+  velox_aggregates
```

Comment:
Is this change needed?

Reply:
Yes, it is not necessary. I can get rid of it now.